Python/AWS testing Script/Readme.md

```diff
@@ -0,0 +1,29 @@
+## Aws Script for testing using moto (a AWS testing framework )
+
+This python script automatically generates a set of EC2 machine's with any demo ami Id that could be used in case of testing.
+
+## Installation
+
+First of all install [python]("https://www.python.org/downloads/") on your system.
+```bash
+pip install moto
+pip install boto3
+pip install Flask
+```
+
+## Results
+
+![](https://i.postimg.cc/Jzx6f02c/image1.png)
+
```

Comment:
Try using Imgur for generating your image links and write it as `![Image](Image_link)`